solace-message-router/azuredeploy.json

```diff
@@ -130,6 +234,55 @@
         }
       }
     },
+    {
+      "apiVersion": "2015-01-01",
+      "type": "Microsoft.Resources/deployments",
+      "name": "[variables('solaceLoadBalancerName')]",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('nestedUrl'), variables('loadBalancerSharedTemplateName'))]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "dnsLabelForLbIp": {
+            "value": "[parameters('dnsLabelForLbIp')]"
+          },
+          "publicIPAddressNameLB": {
+            "value": "[variables('publicIPAddressNameLB')]"
+          },
+          "publicIPAddressType": {
+            "value": "[variables('publicIPAddressType')]"
+          },
+          "lbName": {
+            "value": "[variables('lbName')]"
+          },
+          "lbPoolName": {
+            "value": "[variables('lbPoolName')]"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2015-01-01",
+      "type": "Microsoft.Resources/deployments",
+      "name": "[variables('solaceWorkspaceName')]",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[if(empty(parameters('workspaceName')), concat(variables('nestedUrl'), variables('workspaceDisabledSharedTemplateName')), concat(variables('nestedUrl'), variables('workspaceEnabledSharedTemplateName')))]",
```

Comment:
remove this pattern and just use the condition property for enable/disable

Reply:
Refactored